src/components/ContactDetails.vue

```diff
@@ -56,6 +59,7 @@
 							autocorrect="off"
 							spellcheck="false"
 							name="fullname"
+							@updateLocalContact="updateLocalContact"
```

Comment:
Error?

Reply:
You are right, forgot to remove that from my trials. Good that you saw that!